### PR TITLE
chore: update google style editorconfig

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by hick209** [comment]
@chenrui333 would you mind adding context on what this change affects/does?

File: docs/editorconfig/.editorconfig-google [hunk 1]
```diff
@@ -6,42 +6,38 @@
...
 
-[{*.kt,*.kts}]
-indent_style = space
-insert_final_newline = true
-max_line_length = 100
-indent_size = 2
-ij_continuation_indent_size = 2
-ij_java_names_count_to_use_import_on_demand = 9999
+[{*.gradle.kts,*.kt,*.kts,*.main.kts}]
+indent_size = 4
```
Comment (nreid260) [inline docs/editorconfig/.editorconfig-google:10]: --google-style indents everything by 2 spaces at a time, so this is almost certainly inconsistent.

**Comment by nreid260** [comment]
I skimmed it, but I don't use IntelliJ, and I'm not sure anyone here that does is using an editorconfig. We enforce that they run ktfmt instead.

**Comment by chenrui333** [comment]
this was a dump from the intellij config

File: docs/editorconfig/.editorconfig-google (hunk 1 above)
Comment (chenrui333) [reply docs/editorconfig/.editorconfig-google:10]: though there is no official style guide in https://github.com/google/styleguide, this is the closest one by google team.
https://developer.android.com/kotlin/style-guide#whitespace

And this is the indentation guide
image

Comment (chenrui333) [reply docs/editorconfig/.editorconfig-google:10]: happy to change to indent_size to 2 though.

Comment (chenrui333) [reply docs/editorconfig/.editorconfig-google:10]: @nreid260 correct me if I miss anything.

Comment (nreid260) [reply docs/editorconfig/.editorconfig-google:10]: Our internal style guide is based on the public Android one you linked to, but with substantial modifications. I don't know if there are plans to publish it, but --google-style exists to enforce it. I admit things could be better.

To be clear, I have no stake in this editorconfig file. As far as I'm concerned, Google users will run ktfmt and accept its output. We have no need for IntelliJ to replicate the style.

**Comment by hick209** [comment]
@chenrui333, any particular problem you are trying to solve here?

Since there's not much information on the PR at the moment I'll close it, but feel free to reopen if you believe this is something we should merge.

**Pull request closed without merging**